src/Testcontainers/Configurations/Modules/Databases/AzuriteTestcontainerConfiguration.cs

```diff
@@ -0,0 +1,279 @@
+namespace DotNet.Testcontainers.Configurations.Modules.Databases
+{
+  using System;
+  using System.Collections.Generic;
+  using DotNet.Testcontainers.Builders;
+  using JetBrains.Annotations;
+
+  public class AzuriteTestcontainerConfiguration : IDisposable
```

Comment:
`[PublicAPI]`

Reply:
Fixed in [bddb8be](https://github.com/testcontainers/testcontainers-dotnet/pull/555/commits/bddb8be4dc91b51a89a704799bbe006d784e4f8c)